src/wagtail_ai/static_src/custom.d.ts

```diff
@@ -1,6 +1,11 @@
 /* eslint-disable no-unused-vars */
 export {};
 
+export interface WagtailAiConfiguration {
+  wagtailAiPrompts: Array<Prompt>;
+  wagtailAiProcessUrl: string;
```

Comment:
**Suggestion:** there's no need to prefix all the settings with `WagtailAi` given the interface already has it in the name.

Reply:
Renamed in f4890e2952f17de4f10326eacf7bef560f7d0a8c.